core/new-gui/src/app/dashboard/component/feature-container/user-project-section/project-page/project-page.component.html

```diff
@@ -0,0 +1,248 @@
+<div> 
```

Comment:
many components here are similar or duplicates to the existing `WorkflowSectionComponent` and `UserFileSectionComponent`. please reuse them to reduce duplicated code, otherwise it's harder to maintain two copies.

Reply:
will reuse WorkflowSectionComponent and UserFileSectionComponent + modify them to accommodate for UserProject functionality in future PR